_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -951,7 +1028,7 @@ terraform {
 
 Next, use the `account-baseline-root` module from the Gruntwork Infrastructure as Code Library, making sure to replace the `<VERSION>` placeholder
 with the latest version from the https://github.com/gruntwork-io/module-security/releases[releases page] (minimum
-version for this guide is `v0.34.1`):
+version for this guide is `v0.36.0`):
```

Comment:
Should we version lock this guide instead of recommending the latest? We're not really doing a good job keeping these in sync with releases, and it may be a more stable experience for customers to always use the version that we tested with.

Reply:
It's a trade-off, as we may get stuck with a stale version here for a long time... But given the flakiness of the LZ stuff, I suppose a known-good version is the lesser of two evils here. Updated: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/354/commits/5ffcdc2173ef069113b39991818b13db276e15ea